ledger/sandbox/src/main/resources/db/migration/h2database/V8__Party_entries.sql

```diff
@@ -0,0 +1,42 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V8: party_entries
+--
+-- This schema version adds a table for tracking party allocation submissions
+---------------------------------------------------------------------------------------------------
+
+CREATE TABLE party_entries
+(
+    -- The ledger end at the time when the party allocation was added
+    ledger_offset    bigint primary key  not null,
+    recorded_at      timestamp           not null, --with timezone
+    -- SubmissionId for the party allocation
+    submission_id    varchar,
+    -- participant id that initiated the allocation request
+    -- may be null for implicit party that has not yet been allocated
+    participant_id   varchar,
+    -- party
+    party            varchar,
+    -- displayName
+    display_name     varchar,
+    -- The type of entry, 'accept' or 'reject' or 'implicit'
+    typ              varchar             not null,
+    -- If the type is 'reject', then the rejection reason is set.
```

Comment:
document meaning of implicit?

Reply:
`implicit` party entries don't exist anymore. I'll remove it. thanks for catching it.

Reply:
Addressed in 9ac958eb4.